commands/anime/moe.js

```diff
@@ -1,6 +1,6 @@
 const Command = require('../../structures/Command');
 const Discord = require('discord.js');
-const randomPuppy = require('random-puppy');
+const snekfetch = require('snekfetch');
```

Comment:
I will try this, and maybe move back to a single request package, so either only axios, only node-superfetch, or only snekfetch.